packages/browser/src/exports.ts

```diff
@@ -56,6 +56,8 @@ export {
   withScope,
   FunctionToString,
   InboundFilters,
+  metrics,
+  Metrics,
```

Comment:
Since this is a type why don't we include with import type statement?

Reply:
`Metrics` is not a type, it is a class.

```ts
Sentry.init({
  dsn: '__DSN__',
  integrations: [
    new Sentry.Metrics(),
  ],
});
```

I do realize it's kind of confusing to have the class be the same name, let me think about how to rename it.